docs/components/iam/getting-started/docker/start-iam.md

```diff
@@ -0,0 +1,33 @@
+---
+id: start-iam
+title: "Start IAM"
+sidebar_label: "Step 2: Start IAM"
+---
+
+### Starting the containers
+To start the containers we can use a couple of methods, you may have Docker support within your IDE, if so please consult their documentation on run configurations.
+If you are using the command line you can use the following global command:
+
+```shell
+docker-compose -f /path/to/your/docker-compose.yml up -d
```

Comment:
Since Docker `3.4` the default command would be `docker compose` I would also use it here as default and add a not that `docker-compose` can be used when running with Docker Compose V1.

```suggestion
docker compose -f /path/to/your/docker-compose.yml up -d
```